### PR TITLE
📝 Update setup steps in README

### PR DESCRIPTION
**Pull request by znatty22**
I went through the README to try and setup the Data Tracker dev env and came across some things. Here's an updated README with some fixes/clarifications. Not sure if all are valid (do we still use kf-data-stack??)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,35 +13,35 @@ Investigator study management and progress tracking.
...
 
 ```
 docker network create kf-data-stack
+docker volume create --name=bundle
 ```
 
-Checkout kf-api-study-creator master and bring up the study creator API:
-Checkout the [Kids First Data Stack](https://github.com/kids-first/kf-data-stack)
-and make sure the study creator services are running:
+Next bring up the study creator API in development mode:
```
Comment (znatty22) [inline README.md:34]: Not really sure what was supposed to go here. Do we use kf-data-stack docker-compose, study-creator docker-compose or data-tracker docker-compose?

Comment (dankolbman) [reply README.md:34]: The docker-compose included in this repo sets up the study creator and all other apis needed for testing. The line you have should work for just bringing up the essential study creator api.

File: README.md [hunk 2]
```diff
@@ -64,6 +64,9 @@ REACT_APP_AUTH0_AUD
 
 See the provided `.env.schema` file for more details.
 
+You may need to contact the project maintainer to get the appropriate value for
```
Comment (dankolbman) [inline README.md:67]: We can replace with with 
> You will need to setup an application in Auth0 and use it to set the `REACT_APP_AUTH0_CLIENT_ID` variable. Make sure to configure the application to allow redirects at the local server's address (typically http://localhost:3000/callback)

Comment (znatty22) [reply README.md:67]: Ok so is it expected that each developer will need to setup an application in Auth0 to get their client ID? Or is this blurb for the general developer outside Kids First? In other words, are all Kids First developers expected to use the same client id?

Comment (dankolbman) [reply README.md:67]: Devs outside Kids First should need to set up their own Auth0 app. Devs in Kids First could use the same application client id or set them up individually.

**Review by dankolbman — APPROVED** [review]
# 🍄

Thanks, we definitely need to do a lot of cleanup with the developer side of things from a better readme to just organizing our config better.

**Pull request merged**